src/main/resources/config/application.yaml

```diff
@@ -40,6 +40,6 @@ powsybl-ws:
     customQuery: ${powsybl-ws.database.customQueryBegin}reWriteBatchedInserts=true
     name: sa
 
-# default values for loadflow providers "Hades2" or "OpenLoadFlow"
+# default values for security-analysis providers "Dynaflow" (SURE ?) or "OpenLoadFlow"
```

Comment:
Only OLF is enough. Security analysis are not available with DynaFlow I think. To be checked

Reply:
Actually it's available. So to be kept

Reply:
So we should allow to use it in study-server ?